waiter/src/waiter/cookie_support.clj

```diff
@@ -59,10 +59,10 @@
 
 (defn add-encoded-cookie
   "Inserts the provided name-value pair as a Set-Cookie header in the response"
-  [response password name value age-in-days]
+  [response password name value age-in-seconds]
   (letfn [(add-cookie-into-response [response]
             (let [encoded-cookie (UrlEncoded/encodeString (encode-cookie value password))
-                  max-age (-> age-in-days t/days t/in-seconds)
+                  max-age age-in-seconds
```

Comment:
this variable is a reassignment and can be removed.

Reply:
done